earthpy/tests/test_plot_rgb.py

```diff
@@ -146,11 +147,15 @@ def test_stretch_output_scaled(rgb_image):
     """
     arr, _ = rgb_image
     stretch_vals = list(range(10))
-    axs = [plot_rgb(arr, stretch=True, str_clip=v)[1] for v in stretch_vals]
-    mean_vals = np.array([ax.get_images()[0].get_array().mean() for ax in axs])
-    n_unique_means = np.unique(mean_vals).shape[0]
-    assert n_unique_means == len(stretch_vals)
+
+    mean_vals = list()
+    for v in stretch_vals:
+        ax = plot_rgb(arr, stretch=True, str_clip=v)[1]
+        mean = ax.get_images()[0].get_array().mean()
+        mean_vals.append(mean)
+        plt.close()
+    assert len(set(mean_vals)) == len(stretch_vals)
     try:
```

Comment:
@lwasser this looks better! Sorry if I'm missing something, but what is this try/finally logic doing at the end of this test? Can we get rid of it? I am not seeing how it's relevant to the assertion that we test against.

Reply:
it's deleting the ax object which remains open post figure close. it's super confusing to me why the ax obj is still open after you close the figure. i guess `plt.cla()` would clear that? does it matter @mbjoseph

Reply:
Because of the way that pytest isolates test environments, the `ax` object and any other objects created in the test should be deleted after the test is finished. 

I just tested locally with deleting the try/except statement, and all the tests still pass with no warning, so I think we should go ahead and delete it @lwasser.

Reply:
ok great. yes it doesn't break anything. it just cleans up ALL elements of the plot object. i'll remove it.